lib/services/project-service.ts

```diff
@@ -111,6 +113,9 @@ export class ProjectService implements IProjectService {
 
 			this.$projectDataService.initialize(projectDir);
 			this.$projectDataService.setValue("id", projectId).wait();
+			//Start child process because --save --save-exact option is necessary when installing tns-core-modules
+			//in order to create "dependencies" key in package.json. 
+			this.$childProcess.exec("npm install " + constants.TNS_CORE_MODULES_NAME + " --save --save-exact", { cwd: projectDir }).wait();
```

Comment:
Should it be better to use `INodePackageManager` and expose this functionality there?